apps/frontend/src/app/Components/StarDisplay.tsx

```diff
@@ -1,10 +1,7 @@
-import { faStar } from '@fortawesome/free-solid-svg-icons'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { Rarity } from '../Types/consts'
-import ColorText from './ColoredText'
-
-export const StarIcon = () => <FontAwesomeIcon icon={faStar} />
+import { Rarity } from '@genshin-optimizer/consts';
+import StarRoundedIcon from '@mui/icons-material/StarRounded';
+import ColorText from './ColoredText';
 export const StarsDisplay = ({ stars, colored = false }: { stars: Rarity, colored?: boolean }) =>
   <ColorText color={colored ? "warning" : undefined} >
-    {stars ? [...Array(stars).keys()].map((_, i) => <StarIcon key={i} />) : null}
+    {stars ? [...Array(stars).keys()].map((_, i) => <StarRoundedIcon key={i} fontSize="inherit" sx={{ verticalAlign: "-10%" }} />) : null}

```

Comment:
These are a bit smaller than they used to be, could we increase the size?
here: ![image](https://user-images.githubusercontent.com/36019388/215015852-e24bd268-7224-4682-addb-53d0e537a67e.png)

prod: ![image](https://user-images.githubusercontent.com/36019388/215015890-6486f95b-df9a-4a5a-ab1d-4d4cfda5c06b.png)